Octokit/Clients/RepositoriesClient.cs

```diff
@@ -482,7 +482,7 @@ public Task<IReadOnlyList<Repository>> GetAllForOrg(string organization, ApiOpti
         /// Client for GitHub's Repository Deployments API
         /// </summary>
         /// <remarks>
-        /// See the <a href="http://developer.github.com/v3/repos/deployment/">Collaborators API documentation</a> for more details
+        /// See the <a href="https://developer.github.com/v3/repos/deployments/">Collaborators API documentation</a> for more details
```

Comment:
It's also still called `Collaborators` in here due to someone copy-pasting things. Could you update that?